src/Framework/test/TestData.cs

```diff
@@ -29,6 +29,7 @@ static TestData()
                 "Microsoft.AspNetCore.Authorization.Policy",
                 "Microsoft.AspNetCore.Components",
                 "Microsoft.AspNetCore.Components.Authorization",
+                "Microsoft.AspNetCore.Components.Endpoints",
```

Comment:
@javiercn Add this to `ListedTargetingPackAssemblies` below as well (around line 177)